apps/server/src/modules/school/domain/do/system.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
+import { SystemProvisioningStrategy } from '@shared/domain/interface/system-provisioning.strategy';
+import { LdapConfig, OauthConfig, OidcConfig } from '../type';
+
+// TODO: There is already a systems module. Question for the future: Should this stay separate or be integrated in the school module?
+export class System extends DomainObject<SystemProps> {}
+
+interface SystemProps extends AuthorizableObject {
+	type: string;
+	url?: string;
+	alias?: string;
+	displayName?: string;
+	oauthConfig?: OauthConfig;
```

Comment:
Ich vermute mal das es zwischen oauthConfig, oidcConfig, ldapConfig eine "entweder oder" Verknüpfung gibt. Das könnte man gut über Methoden abbilden die beim neuzuweisen die anderen beiden Felder auf undefined setzten und ein get Methode die dann das definierte von den 3 ausliefert, oder undefined. 
Ist aber auch ein seperates Thema, daher eher Teil eines Cleanup Tickets nachdem das legacySchool Module nicht mehr existiert und man gefahrlos Anpassungen in den Clients bzg. Response Key machen kann.

Reply:
Hab die Systems betreffenden Sachen aus dem PR genommen, siehe anderen Kommentar.